src/main/java/com/onegini/examples/resourcegateway/ResourceGatewayApplication.java

```diff
@@ -1,52 +1,34 @@
 package com.onegini.examples.resourcegateway;
 
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.context.annotation.Bean;
-import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
-import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
 import org.springframework.web.client.RestTemplate;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.PropertyNamingStrategy;
 
 @SpringBootApplication
 public class ResourceGatewayApplication {
 
-  private static final int TIMEOUT_IN_MILLIS = 3000;
-
   public static void main(final String[] args) {
     SpringApplication.run(ResourceGatewayApplication.class, args);
   }
 
-  @Autowired
-  private ObjectMapper objectMapper;
-
-  @Autowired
-  public void configureJackson(final ObjectMapper jackson2ObjectMapper) {
-    jackson2ObjectMapper.setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE);
-    jackson2ObjectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
-  }
-
   @Bean
-  public MappingJackson2HttpMessageConverter mappingJackson2HttpMessageConverter() {
-    MappingJackson2HttpMessageConverter converter = new MappingJackson2HttpMessageConverter();
-    converter.setObjectMapper(objectMapper);
-    return converter;
+  public Jackson2ObjectMapperBuilder jacksonBuilder() {
```

Comment:
Nice improvement! It's not needed to configure the message converter explicitly in case we just want to modify the properties of the default object mapper 👍 .

Reply:
Indeed and then we get everything from Spring Boot's configuration for free